client/my-sites/domains/domain-management/gsuite/gsuite-add-users/add-another-user-link.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './add-users.scss';
+
+class AddAnotherUserLink extends React.Component {
+	handleAddUserClick = event => {
+		event.preventDefault();
+		this.props.recordAddUserClick( this.props.domain );
+		this.props.addBlankUser();
+	};
+
+	render() {
+		return (
+			<button
+				type="button"
+				className="gsuite-add-users__add-another-user-link"
+				onClick={ this.handleAddUserClick }
+			>
+				{ this.props.translate( '+ Add another user' ) }
```

Comment:
We should pull out the `+` here so it is not a separate translation from the one the bot found.